sound/soc/codecs/rt1308-sdw.c

```diff
@@ -633,6 +675,13 @@ static struct snd_soc_dai_driver rt1308_sdw_dai[] = {
 			.rates = RT1308_STEREO_RATES,
 			.formats = RT1308_FORMATS,
 		},
+		.capture = {
+			.stream_name = "DP2-4 Capture",
+			.channels_min = 2,
+			.channels_max = 2,
```

Comment:
actually @shumingfan should this be channel_max = 4? Or should we remove channels_max if it's irrelevant?

Reply:
> actually @shumingfan should this be channel_max = 4? Or should we remove channels_max if it's irrelevant?
@plbossart Sorry, I missed this part. I changed .channels_min/.channels_max to 4 when I tested the feedback.

Reply:
no worries, changing it now.